roles/servicetelemetry/templates/manifest_prometheus.j2

```diff
@@ -42,10 +41,11 @@ spec:
     - -tls-cert=/etc/tls/private/tls.crt
     - -tls-key=/etc/tls/private/tls.key
     - -upstream=http://localhost:9090/
-    - -htpasswd-file=/etc/proxy/htpasswd/auth
     - -cookie-secret-file=/etc/proxy/secrets/session_secret
     - -openshift-service-account=prometheus-stf
-    - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
+    - '-openshift-sar={"namespace":"{{ ansible_operator_meta.namespace }}","resource": "prometheus", "group":"{{ prometheus_operator_api_string | replace("/v1","") }}", "verb":"get"}'
```

Comment:
I'd like to change the other SARs in the system to be more specific like this, too. The existing one matched what we found in openshift-monitoring when we were headed in that direction, and is a rough approximation of "admin user" without referencing a specific admin group. Using more specific check like this makes more sense to me. This is probably release-note worthy.

Reply:
https://github.com/infrawatch/service-telemetry-operator/pull/553
https://github.com/infrawatch/smart-gateway-operator/pull/154